uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -613,11 +613,11 @@ private function createAbsenceType() {
     ]);
   }
 
-  private function createPeriodEntitlement() {
+  private function createPeriodEntitlement($period_id = 1, $type_id = 1, $contact_id = 1) {
```

Comment:
Can you please remove this method completely and create a fabricator for LeavePeriodEntitlement?